nltk/internals.py

```diff
@@ -523,6 +529,12 @@ def find_file(filename, env_vars=(), searchpath=(),
     return next(find_file_iter(filename, env_vars, searchpath,
                                file_names, url, verbose))
 
+
+def find_dir(filename, env_vars=(), searchpath=(),
+        file_names=None, url=None, verbose=True):
+    return next(find_file_iter(filename, env_vars, searchpath,
+                               file_names, url, verbose, finding_dir=True))
+
```

Comment:
Please add a new line, as between the top level functions there should be 2 newlines.